src/components/ParsedHtml/style.css

```diff
@@ -0,0 +1,15 @@
+.parsed-link {
+    text-decoration: none;
+}
+
+body.chakra-ui-dark .parsed-link {
+    color: #7F99FB !important;
+}
+
+body.chakra-ui-light .parsed-link {
+    color: #2D4EC9 !important;
+}
```

Comment:
Since the selector is now way more precise, maybe the `!important` is not needed anymore, but I'm not sure at all as I didn't test why it was there, this is probably here only because there is already an important on that link

Approved as it's not blocking, thanks :)